src/components/Strategies/StrategyList.tsx

```diff
@@ -85,8 +66,24 @@ const Container = styled.div`
     display: inline-block;
   }
 
-  .soon {
+  .comingSoon {
     float: right;
+    animation: opacity 0.5s ease;
+  }
+
+  .visible {
```

Comment:
Another thing - since `.visible` applies only to `.comingSoon` then move it inside `.comingSoon` (you can nest CSS here)